common/src/events/event_types/note/inner.rs

```diff
@@ -5,6 +5,17 @@ use clap_sys::events::clap_event_note;
 use std::fmt::Formatter;
 use std::marker::PhantomData;
 
+/// Generic CLAP note event.
+///
+/// Notes and voices in CLAP are addressed by a 4‑value tuple
+/// `(port, channel, key, note_id)`, called a PCKN in the CLAP headers.
+///
```

Comment:
Small nitpick: the PCKN concept is not just in the CLAP headers, it's condensed into the Pckn type in Clack :)

Reply:
Ah, yeah I see what you mean. `Pckn` is actually the abstraction that's meant to be used to target notes in events (although it's optional like most things in Clack). It also enforces checking for the wildcard case, which is represented by the `Match` enum instead by a `-1` value.

But yes, just referencing it would be enough I think! :slightly_smiling_face: